### PR TITLE
Added SMTP_METHOD variable to set delivery_method as smtp or async_smtp

### PR DESCRIPTION
**Pull request by fbertux**
Hi!

I created an variable to set smtp method as smtp or async_smtp. I think that is an option to set email as sendmail (issue 72). I don't know if this solution it's correct, but for me works :D 
Do I need to change anything else?

Regards

### CODE REVIEW

File: assets/init [hunk 1]
```diff
@@ -369,6 +370,12 @@ if [ "${SMTP_ENABLED}" == "true" ]; then
     "") sudo -u redmine -H sed '/{{SMTP_AUTHENTICATION}}/d' -i config/initializers/smtp_settings.rb ;;
     *) sudo -u redmine -H sed 's/{{SMTP_AUTHENTICATION}}/'"${SMTP_AUTHENTICATION}"'/' -i config/initializers/smtp_settings.rb ;;
   esac
+
+  case "${SMTP_METHOD}" in
```
Comment (sameersbn) [inline assets/init]: @fbertux this else condition is not required. simply add the `
*) sudo -u redmine -H sed 's/{{SMTP_AUTHENTICATION}}/'"${SMTP_AUTHENTICATION}"'/g' -i config/initializers/smtp_settings.rb` line at https://github.com/sameersbn/docker-redmine/blob/master/assets/init#L346

Notice that i have add the `g` option at the end of your `sed` command. It is required when trying to change more than matches in `sed` which in your case are two instances of the string `{{SMTP_METHOD}}`.

Please update the PR and let me know. Please make sure you test it out before updating the PR. Thanks.

**Comment by sameersbn** [comment]
@fbertux also please mention the `SMTP_METHOD` setting in the readme, with the possible settings

**Comment by fbertux** [comment]
I changed the lines that you mentioned and I added a description on README.

**Comment by frank-dspeed** [comment]
this pull closes #72

**Pull request merged**